examples/complete/variables.tf

```diff
@@ -6,4 +6,10 @@ variable "allow_ssl_requests_only" {
   type        = bool
   default     = true
   description = "Set to `true` to require requests to use Secure Socket Layer (HTTPS/SSL). This will explicitly deny access to HTTP requests"
-}
\ No newline at end of file
+}
+
+variable "lifecycle_configuration_rules" {
+  type        = any
```

Comment:
should it be `list(any)` ?

Reply:
Really it should match the module's variable definition, but I didn't want to have to keep them in sync.